src/Components/ClustersListTable/ClustersListTable.js

```diff
@@ -137,23 +145,32 @@ const ClustersListTable = ({
     }
   }, [filters, filterBuilding]);
 
-  const buildFilteredRows = async (items) => {
-    setRowsFiltered(false);
+  const buildFilteredRows = async (items, signal) => {
     const filtered = items.filter((it) => {
       return passFiltersCluster(it, filters);
     });
     const clusterArr = filtered.map((cluster) => cluster.cluster_id);
     let upgradeArr = [];
     if (clusterArr.length > 0) {
       try {
-        const upgradeRisks = await getClustersUpgradeRisks({
-          clusters: clusterArr,
-        }).unwrap();
-        if (upgradeRisks.status === 'ok') {
-          upgradeArr = upgradeRisks.predictions;
+        const res = await axios.post(
+          '/api/insights-results-aggregator/v2/upgrade-risks-prediction',
+          { clusters: clusterArr },
+          {
+            signal,
+          }
+        );
```

Comment:
Removed the lazy query and replaced it with a bare axios call, needed to insert the signal which is used to cancel the request